### ISSUE TITLE
AWSSQSSource deletes messages in case of sink error

### ISSUE THREAD

**Issue by Kaiass** [triggermesh/triggermesh#1199]
AWSSQSSource probably has a bug which results in deleting messages from queue even if sink returned error.
The bug is in following code [source](https://github.com/triggermesh/triggermesh/blob/main/pkg/sources/adapter/awssqssource/process.go)

```
			for _, event := range events {
				if err := sendSQSEvent(ctx, a.ceClient, event); err != nil {
					a.logger.Errorw("Failed to send event to the sink", zap.Error(err))
					continue
				}
			}
```
Here 'continue' operation refers to the inner loop but seems like should refer to the outer loop.

I have no rights to push branches to Git repository so cannot create a pull request.
How can I get an access?

**Comment by tzununbekov** [comment]
Hi @Kaiass, nice catch!
You should be able to fork the repository, commit your changes to your fork, and then open PR from it back to the upstream.

**Comment by Kaiass** [comment]
Hello @tzununbekov !

Thanks for advice, I've created pull request.

**Comment by tzununbekov** [comment]
Resolved in #1201.
Thank you for your contribution.

**Comment by Kaiass** [comment]
@tzununbekov Thanks for support!
